### PR TITLE
feat (Taiga 1787 | Licence ): created the step and updated the file templator with mappings

### PR DESCRIPTION
**Pull request by OwenGalvia**
Currently works for the following mappings:

Licence Name: <Licence Name>
Application ID: <Application ID>
System Reference Number: <System Reference Number>
Planning Reference Number: <Planning Reference Number>
CM Reference: <CM Reference>
Licence Number: <Licence Number>

Licence Contacts: <Licence Contacts>
Licence CM: <Licence CM>
Licence Request Date: <Licence Request Date>
Licence Reported Date: <Licence Reported Date>
Licence Acknowledged Date: <Licence Acknowledged Date>
Licence Submitted Date: <Licence Submitted Date>
Licence Issued Date n1: <Licence Issued Date n1>
Licence Issued Date: <Licence Issued Date>

Proposed Start Date: <Proposed Start Date n1>
Actual Start Date: <Actual Start Date>
Actual End Date: <Actual End Date>
Valid Until: <Valid Until>
Classification Date: <Classification Date>

Applicant: <Applicant>
New Licensee: <New Licensee>
Files: <Files>

Grade E: <Grade E>
Grade D: <Grade D>

Recipient Name: <Recipient Name>
Recipient Courtesy Title: <Recipient Courtesy Title>
Recipient Email: <Recipient Email>

Applicant Courtesy Title: <Applicant Courtesy Title>
Applicant Name: <Applicant Name>
Applicant Email: <Applicant Email>
Applicant Contact Point: <Applicant Contact Point>
Applicant County: <Applicant County>
Applicant Postcode: <Applicant Postcode>
Applicant Address: <Applicant Address>

Grade D Courtesy Title: <Grade D Courtesy Title>
Grade D Email: <Grade D Email>
Grade D County: <Grade D County>
Grade D Postcode: <Grade D Postcode>
Grade D Address: <Grade D Address>

Grade E Courtesy Title: <Grade E Courtesy Title>
Grade E Email: <Grade E Email>
Grade E County: <Grade E County>
Grade E Postcode: <Grade E Postcode>
Grade E Address: <Grade E Address>

Company Name: <Company Name>
Company Correspondence Name: <Company Correspondence Name>
Company Email: <Company Email>
Company County: <Company County>
Company Postcode: <Company Postcode>
Company Building Name: <Company Building Name>
Company Building Number: <Company Building Number>
Company Street: <Company Street>
Company Locality: <Company Locality>
Company Address: <Company Address>

Applicant Courtesy Title: <Applicant Courtesy Title>
Applicant Email: <Applicant Email>
Applicant County: <Applicant County>
Applicant Postcode: <Applicant Postcode>
Applicant Address: <Applicant Address>

Associated Activities: <Associated Activities>
Below is from the Activity Model:
Townland: <Townland>
Council: <Council>
Irish Grid Reference: <Irish Grid Reference>
Location Description: <Location Description>

### CODE REVIEW

File: coral/media/js/views/components/plugins/licensing-workflow.js [hunk 1]
```diff
@@ -412,26 +413,54 @@ define([
...
+                  componentName: "file-template",
+                  uniqueInstanceName: "letter-template"
                 }
               ]
             }
-          ]
+          ],
+          workflowstepclass: "workflow-form-component"
         },
+        // {
```
Comment (taylorn01) [inline coral/media/js/views/components/plugins/licensing-workflow.js:443]: Can commented code that won't be used removed

File: coral/views/file_template.py [hunk 2]
```diff
@@ -277,107 +282,115 @@ def insert_custom(self, document, k, v, config=None):
...
-        }
-        activity_system_ref_tile = Tile.objects.filter(**activity_system_ref_query).exclude(
-            nodegroup_id=self.ACTIVITY_SYSTEM_REF_NODEGROUP,
-            resourceinstance_id=self.resource_instance.resourceinstanceid,
-        ).first()
-        activity_resource = Resource.objects.filter(
-            pk=activity_system_ref_tile.resourceinstance.resourceinstanceid
-        ).first()
-        return activity_resource
+#     def __init__(self, resource_instance):
```
Comment (taylorn01) [inline coral/views/file_template.py:308]: Same here for removing commented code if its not being used

**Review by taylorn01 — CHANGES_REQUESTED** [review]

File: coral/views/file_template.py (hunk 2 above)
Comment (OwenGalvia) [reply coral/views/file_template.py]: removed

File: coral/media/js/views/components/plugins/licensing-workflow.js (hunk 1 above)
Comment (OwenGalvia) [reply coral/media/js/views/components/plugins/licensing-workflow.js]: removed

**Pull request merged**